proposals/0315-Add-RPC-Conflict-Management.md

```diff
@@ -190,6 +191,7 @@ Also, below shows the Json example for the `rpc_priority_table` adding in policy
     },
     "app_priority":{
         "EMERGENCY": 0,
+        "_comment":"// Does not need to be included, if it is included it will be ignored no matter its assigned priority. See above.",
```

Comment:
```suggestion
    “EMERGENCY”: 0, // Does not need to be included, if it is included it will be ignored no matter its assigned priority. See above.
```